### PR TITLE
Claim rewards for all positions directly from the pools page

### PR DESCRIPTION
**Pull request by nicolaslara**
### ClickUp Task

[ClickUp Task URL](PASTE_CLICKUP_TASK_URL_HERE)

## Summary by CodeRabbit

- **New Features**
	- Introduced a feature to send rewards messages for all positions within the Osmosis account.
	- Added a "Claim All Rewards" button component for users to easily claim rewards from Osmosis pools.
	- Integrated feature flag for concentrated liquidity, enhancing future feature rollouts and testing.

- **Refactor**
	- Enhanced liquidity position querying capabilities in the Osmosis account implementation.

### CODE REVIEW

File: packages/web/components/buttons/claim-all-rewards.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  // TODO: Use translations
+  //const {t} = useTranslation();
+  return (
+    <div className="max-w-xs">
+      <Button
+        mode="secondary"
+        className="mt-8 gap-3 whitespace-nowrap !px-6"
+        onClick={() => onToggle(false)}
+      >
+        Claim All Rewards
```
Comment (jonator) [inline packages/web/components/buttons/claim-all-rewards.tsx:18]: FYI it's possible to add a key to en.json (somewhere that makes sense) and run `yarn machine-translate && yarn lint:fix` in the root of the repo to generate translations

**Comment by MaxMillington** [comment]
@nicolaslara is this still being worked on?

**Comment by nicolaslara** [comment]
No. Worked on this in NJ and should be ready for review / merge. The only thing left should be deciding where to position the button

**Comment by MaxMillington** [comment]
@nicolaslara can you sync with Nass on this? I recommend making a proposal for him to review.

**Comment by nicolaslara** [comment]
Reopened. 

We probably want to move the button up. It would also be good for someone from the FE team to take a look at the code. This made sense in Nov when I wrote it, but a lot of things have changed in the FE architecture since then


Cursor_and_Gather_Mini_Mode

**Comment by jonator** [comment]
closing as stale, can reopen, and could use some design input

**Pull request closed without merging**